calplus/tests/unit/test_client.py

```diff
@@ -4,11 +4,31 @@
 from calplus import client
 from calplus import exceptions
 from calplus.tests.base import TestCase
+from calplus.utils import pick_host_with_specific_provider
 # from calplus.v1.network import client as network_client
 
 CONF = calplus.conf.CONF
 
 
+fake_aws_host = {
+    'aws_access_key_id': 'fake_id',
+    'aws_secret_access_key': 'fake_key',
+    'region_name': 'RegionOne',
+    'endpoint_url': 'http://localhost:35357/v3/',
+    'limit': {
+        "subnet": 10,
+        "vpc": 5,
+        "floatingip": 50,
+        "subnetpool": -1,
+        "security_group_rule": 100,
+        "security_group": 10,
+        "router": 10,
+        "rbac_policy": -1,
+        "port": 50
+    }
+}
+
+
 class TestClient(TestCase):
 
```

Comment:
Please add new unit test for multiple cloud providers case.
E.g: create 3 fake provider: openstack1, openstack2 and aws, then assert CAL can realize all 3 providers with correct configs.